src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/NotificationService.kt

```diff
@@ -32,55 +32,105 @@ class NotificationService(
         @Value("\${application.to.defaultMonths}") val monthStep: Long,
         val notifyClient: NotificationClientApi,
         val prisonService: PrisonService,
-        val csrClient: CsrClient
-) {
+        val csrClient: CsrClient) {
 
-    fun getNotifications(processOnReadParam: Optional<Boolean>, unprocessedOnlyParam: Optional<Boolean>, fromParam: Optional<LocalDate>, toParam: Optional<LocalDate>): Collection<NotificationDto> {
-        val start = calculateStartDateTime(fromParam, toParam)
-        val end = calculateEndDateTime(toParam, start)
-        return getShiftNotificationDtos(start, end, unprocessedOnlyParam.orElse(false), processOnReadParam.orElse(true))
+    fun getNotifications(processOnReadParam: Optional<Boolean>, unprocessedOnlyParam: Optional<Boolean>, fromParam: Optional<LocalDate>, toParam: Optional<LocalDate>, quantumId: String = authenticationFacade.currentUsername): Collection<NotificationDto> {
+        val from = calculateStartDateTime(fromParam, toParam)
+        val to = calculateEndDateTime(toParam, from)
+        val unprocessedOnly = unprocessedOnlyParam.orElse(false)
```

Comment:
What's going on here mate? Is this to avoid an NPE?

Reply:
see Slack :)